src/themes/rhdh/components.ts

```diff
@@ -97,6 +97,20 @@ export const components = (themeConfig: ThemeConfig): Components => {
     };
   }
 
+  // MUI AppBar
+  if (options.headers !== "mui") {
```

Comment:
I think we should use the MUI component name here to align it with the other options:

```suggestion
  if (options.appbar !== "mui") {
```